pkg/api/controllers/attachment_test.go

```diff
@@ -46,6 +47,84 @@ func init() {
 //                         Tests for volume attachment                          //
 ////////////////////////////////////////////////////////////////////////////////
 
+func TestCreateVolumeAttachment(t *testing.T) {
+	var jsonStr = []byte(`{
+		"id": "f2dda3d2-bf79-11e7-8665-f750b088f63e",
+		"name": "fake volume attachment",
+		"description": "fake volume attachment",
+		"hostId": "202964b5-8e73-46fd-b41b-a8e403f3c30b",
+		"volumeId": "bd5b12a8-a101-11e7-941e-d77981b584d8",
+		"attachMode": "ro"
+	}`)
+	var expectedJson = []byte(`{
```

Comment:
Is this required?

Reply:
You are just using jsonSTR

Reply:
yes to convert in byte reader format

Reply:
Ok but it is just used for Print and nothing else, right? Sorry if i missed anything

Reply:
Yes, but a cleaner way to provide request body